public/analyze.js

```diff
@@ -212,16 +225,24 @@ class Watson{
             numberOfStars++;
           }
           reviewsCont.innerHTML += '<div class="stars-inner fa">' + starsContent + '</div></div>';
-          
+
 
           reviewsCont.innerHTML += '<p id = "reviewText">' + '<p><b>' +
-            results[i].title + ' - ' + ' </b>' + results[i].text + '<br></p>';   
+            results[i].title + ' - ' + ' </b>' + results[i].text + '<br></p>';
         }
 
         var posPercent = Math.round(100*(posCount / reviewLen));
         var negPercent = Math.round(100*(negCount / reviewLen));
         var neuPercent = Math.round(100*(neuCount / reviewLen));
 
+        var sentimentWeightedScore = (sentimentWeightedScoreSum / reviewLen);
```

Comment:
and here

Reply:
Yup. Did this happen since we both did a pull request on the same file around the same time?